index.js

```diff
@@ -288,6 +289,13 @@ const DEFAULTS = {
   plugins: [],
   printFunctionName: true,
   escapeRegex: false,
+  highlight: false,
+  theme: {
+    tag: 'cyan',
+    content: 'white',
```

Comment:
You should never use `white` only `reset` since some people have white backgrounds